src/utils/http.js

```diff
@@ -0,0 +1,33 @@
+import {
+  get as rawGet,
+  post as rawPost,
+  patch as rawPatch,
+  del as rawDelete,
+} from 'axios';
+import { merge } from 'lodash/fp';
+import { LOGIN_URL } from '../config/routing';
+import { LOGIN_ENDPOINT } from '../config/endpoints';
+
+const handleUnAuthorizedError = (error) => {
+  const { response: { status, config: { url } } } = error;
+  if (status === 401 && url !== LOGIN_ENDPOINT) window.location = LOGIN_URL;
```

Comment:
window.location reloads the whole page
Use history.push instead

Reply:
fixed